packages/phone-number-privacy/combiner/src/common/io.ts

```diff
@@ -76,9 +78,6 @@ export abstract class IO<R extends OdisRequest> {
   }
 
   requestHasSupportedKeyVersion(request: Request<{}, {}, OdisRequest>, logger: Logger): boolean {
-    if (!requestHasValidKeyVersion(request, logger)) {
```

Comment:
can you add a comment here (or somewhere else) clarifying that an invalid key version defaults to the current version but that an unsupported (i.e. int too high) throws an error?